jobs/env/pull-kubernetes-kubemark-e2e-gce.env

```diff
@@ -1,2 +0,0 @@
-### cluster-env
```

Comment:
Have you deleted this file or just the lines?

Reply:
whole file